Makefile

```diff
@@ -302,8 +302,10 @@ DOCKER_GO = _() { $(SET_X); mkdir -p $(CURDIR)/.go/src/$${3:-dummy} ; mkdir -p $
 
 PARSE_PKGS=$(if $(strip $(EVE_HASH)),EVE_HASH=)$(EVE_HASH) DOCKER_ARCH_TAG=$(DOCKER_ARCH_TAG) KERNEL_TAG=$(KERNEL_TAG) ./tools/parse-pkgs.sh
 LINUXKIT=$(BUILDTOOLS_BIN)/linuxkit
-LINUXKIT_VERSION=e6b0ae05eb3a2b99e84d9ffc03a3a5c9c3e7e371
-LINUXKIT_SOURCE=https://github.com/linuxkit/linuxkit.git
+# linuxkit version. This **must** be a published semver version so it can be downloaded already compiled from
+# the release page at https://github.com/linuxkit/linuxkit/releases
+LINUXKIT_VERSION=v1.3.0
+LINUXKIT_SOURCE=https://github.com/linuxkit/linuxkit
 LINUXKIT_OPTS=$(if $(strip $(EVE_HASH)),--hash) $(EVE_HASH) $(if $(strip $(EVE_REL)),--release) $(EVE_REL)
 LINUXKIT_PKG_TARGET=build
 LINUXKIT_PATCHES_DIR=tools/linuxkit/patches
```

Comment:
Do we plan to use patches (again?)? As I understand we want to stabilise upstream instead, correct? BTW we can move to that case if required in future.

Reply:
We do not, and we have not for a while. We should try as much as we can to use off-the-shelf upstream components.

Reply:
If so, let's delete `LINUXKIT_PATCHES_DIR`.
And please take a look onto the workflow log (I mentioned it in the review comment), 40000 of lines looks sub-optimal.

Reply:
> If so, let's delete LINUXKIT_PATCHES_DIR.

Sure, good idea.

> And please take a look onto the workflow log (I mentioned it in the review comment), 40000 of lines looks sub-optimal

Yeah, separate PR. It has been a problem for a bit. Will clean it up.

Reply:
> If so, let's delete LINUXKIT_PATCHES_DIR.

Done

Reply:
And see #4082 for suppressing those warnings.